site/docs/master/install-overview.md

```diff
@@ -63,6 +63,18 @@ kubectl delete namespace/velero clusterrolebinding/velero
 kubectl delete crds -l component=velero
 ```
 
+## Installing with the Helm chart
+
+When installing using the Helm chart, the provider's credential information will need to be appended into your values.
+
+The easiest way to do this is with the `--set-file` argument, available in Helm 2.10 and higher.
+
+```bash
+helm install --set-file credentials.secretContents.cloud=./credentials-velero
+```
+
+See your cloud provider's documentation for the contents and creation of the `credentials-velero` file.
```

Comment:
I found this sentence slightly confusing the first time I read it - maybe something like "See the Velero installation instructions for your cloud provider for details on how to create the `credentials-velero` file" would be a little more clear?